lib/util.js

```diff
@@ -91,9 +91,13 @@ function errForLog(err) {
  * @return {!string} the generated request ID
  */
 function generateRequestId() {
-
-    return uuid.TimeUuid.now().toString();
-
+    const options = {
```

Comment:
None of these options are actually needed. Le'ts remove them.

Reply:
Removed the options